### ISSUE TITLE
update issues

### ISSUE THREAD

**Issue by witchyyy** [trv6/Lua#2]
Since the new update this mod isn't working correctly. I love it. I hope you can repair it. THanks!!

**Comment by trv6** [comment]
Can you specify which addon you're referencing and add some detail about what isn't functioning?

**Comment by witchyyy** [comment]
Yes sorry its the one for curing without having to head hunt in FFXI

**Comment by witchyyy** [comment]
When in a party it doesn't show the party members at all. I left a party, rejoined a new party and the old members names stayed up - couldn't use it.

**Comment by witchyyy** [comment]
oh and lots of red ERROR messages kept popping up on the screen. I tried to screen shot but I didn't get it. :(

**Comment by witchyyy** [comment]
I called trusts, and they were all ??? (no names)

**Comment by trv6** [comment]
The error with trusts is mostly laziness. The server doesn't send the same packets for trusts, including the packet containing name information and the packet received after zoning while in a party (if you call trusts and then zone without dismissing them, the display will still contain the trusts after zoning). It is possible to pull the name information out of a different packet and do a manual check for party structure after zoning, but the "refresh" command works as well. 
'''//nos r -- after summoning trusts or zoning without releasing'''

Regarding the errors/failure to rebuild the display: That hasn't happened to me in a long time. Could you try this version and let me know if they continue? I know there were a few minor glitches that I patched locally and never pushed. [https://www.dropbox.com/s/g5uer6cy9g3lkic/Nostrum.7z?dl=0](url)

**Comment by witchyyy** [comment]
ahh ok. I am getting an error code with your link :(

**Comment by witchyyy** [comment]
I use windower 4.0 and I'm also not sure how to incorporate that link to the windower.  (not as smart as you obviously lol)

**Comment by witchyyy** [comment]
AHH the //nos r worked!! THANKS!!

**Comment by trv6** [comment]
Sorry, fixed the link. Was the error occurring before the update?

If the file doesn't fix the problem, I will need the actual error from windower's console.

**Comment by witchyyy** [comment]
no it wasn't before the update. The refresh seems to fix it with trusts. I haven't tried in a party yet today.
